client/config.go

```diff
@@ -73,6 +82,14 @@ type DNSResolverConfig struct {
 	Port     string
 }
 
+// ICMP is the configuration for the ICMP client specific config
+type Icmp struct {
+	// Don't Fragment flag (DF) for the client
+	Df bool `yaml:"df"`
+	// Size of the packet
+	Size int `yaml:"size"`
+}
```

Comment:
```suggestion
type ICMPConfig struct {
	DF bool `yaml:"df"`    // Don't Fragment flag used for the ICMP client
	Size int `yaml:"size"` // Size of the packet
}
```
To match other configurations, `Icmp` should be renamed to `ICMPConfig`. 

`Df` should also be refactored to `DF` so that it follows Go's standard naming convention.